### DISCUSSION TITLE
Did the newest update wipe anyone else's data and settings?

### DISCUSSION THREAD

**Discussion by tetra-archos** [hargata/lubelog#506] [Q&A]
I have my static data paths set and the newest update still completely reset my entire installation, did that happen to anyone else before I submit a bug?

**Comment by Judman** [comment]
So I am running 1.2.5 because any time I change a the version and pull new containers, the app starts completely blank. I have to roll back my proxmox backup because even rolling back to 1.2.5 does not fix the install. I do not know what I am doing wrong. My compose file matches the base compose file in the docs.

**Comment by contrerasp** [comment]
Yeah, I haven't a way to update yet. I wish there was an update script to pull latest release from GitHub automatically. Possibly a new feature request?

**Comment by PatrickS7** [comment]
I use the [Community Helper/TTech scripts](https://community-scripts.github.io/ProxmoxVE/scripts?id=lubelogger) to update/install. 

The [documentation ](https://docs.lubelogger.com/Installation/Troubleshooting) says: "_You didn't persist your docker volumes for, check your docker-compose file. All your data should still be there however, they might just be anonymous volumes that you have to re-link back to the container_." 

 My data appears to have been wiped out today with running the update. I am looking into how to fix it.

**Reply by hargata** [reply]
The LXC Script utilizes the Linux Standalone executable instead of the docker image, so the troubleshooting steps for Docker won't apply to you. I'd check if there is a `data` folder in your lubelogger directory with `cartracker.db` file in it, unless you're using Postgres DB. Additional directories to check: `<lubeloggerdirectory>/wwwroot/images` and `<lubeloggerdirectory>wwwroot/documents`

I'm very curious if the unzip command deletes folders that aren't in the archive.

**Reply by PatrickS7** [reply]
Thanks for the reply. That is what I'm finding so far. I did stumble upon the cartracker.db file at /opt/lubelogger/data

I found /opt/lubelogger/wwwroot/documents doesn't appear to exist, nor does /opt/lubelogger/wwwroot/images

**Reply by hargata** [reply]
those folders only exist if you have uploaded attachments to any records or if you have thumbnails for your vehicles.

**Reply by PatrickS7** [reply]
Oh ok. I did have images for the vehicles but, I don't recall adding any attachments. This isn't a huge loss of data, I think I only lost one fuel record, that I can easily re-enter. Hoping to help others avoid data loss. To test, I rolled back to a Proxmox backup of the container from 12/29 and then ran the update again and it seems to wipe it out again. Not sure what part of the process is making it happen.

**Reply by hargata** [reply]
I'm not familiar with LXC, but I can think of one possibility where data can be erased. When the archive of the new version is being unzipped, it is deleting folders that are not in the archive.

The following files and folders are created on runtime and hence won't be in the archive:

- `config\UserConfig.json`
- `data\cartracker.db`
- `wwwroot\images`
- `wwwroot\documents`
- `wwwroot\translations`

If the script is deleting those folders, that would probably explain it.

**Reply by PatrickS7** [reply]
Thanks! I will see what I can find.
